project/src/test/java/com/google/EventTest.java

```diff
@@ -0,0 +1,94 @@
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class EventTest {
+    private static final String HOST_NAME = "Bob Smith";
+    private static final String EMAIL = "bobsmith@google.com";
+    private static final User HOST = new User.Builder(HOST_NAME, EMAIL).build();
+    private static final String EVENT_NAME = "Team Meeting";
+    private static final String NEW_EVENT_NAME = "Daily Team Meeting";
+    private static final String DESCRIPTION = "Daily Team Sync";
+    private static final Set <String> LABELS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList("Tech", "Work")));
+    private static final String LOCATION = "Remote";
+    private static final LocalDate DATE = LocalDate.of(2020, 7, 24);
+    private static final String VOLUNTEER_OPPORTUNITY_NAME = "Volunteer";
+    private static final int NUM_SPOTS_LEFT = 5;
+    private static final VolunteeringOpportunity OPPORTUNITY = new VolunteeringOpportunity.Builder(VOLUNTEER_OPPORTUNITY_NAME, NUM_SPOTS_LEFT).build();
+    private static final Set <VolunteeringOpportunity> OPPORTUNITIES = new HashSet<VolunteeringOpportunity> (Arrays.asList(OPPORTUNITY));
+    private static final User USER1 = new User.Builder("USER1", "USER1@test.com").build();
+    private static final User USER2 = new User.Builder("USER2", "USER2@test.com").build();
+    private static final User USER3 = new User.Builder("USER3", "USER3@test.com").build();
+    private static final Set <User> ATTENDEES = new HashSet<User> (Arrays.asList(USER1, USER2));
+    private static final Set <User> NEW_ATTENDEES = new HashSet<User> (Arrays.asList(USER1, USER2, USER3));
+
+    // User creating event
+	// Verify Builder class is created with correct required and optional fields
+	@Test
+	public void getEventAfterBuild() {
+        Event event = new Event.Builder(EVENT_NAME, DESCRIPTION, LABELS, LOCATION, DATE, HOST).setOpportunities(OPPORTUNITIES).setAttendees(ATTENDEES).build();
+        String actual_name = event.getName();
+        String expected_name = EVENT_NAME;
+        String actual_description = event.getDescription();
+        String expected_description = DESCRIPTION;
+        Set<String> actual_labels = event.getLabels();
+        Set<String> expected_labels = LABELS;
+        String actual_location = event.getLocation();
+        String expected_location = LOCATION;
+        LocalDate actual_date = event.getDate();
+        LocalDate expected_date = DATE;
+        User actual_host = event.getHost();
+        User expected_host = HOST;
+        Set<VolunteeringOpportunity> actual_opportunities = event.getOpportunities();
+        Set<VolunteeringOpportunity> expected_opportunities = OPPORTUNITIES;
+        Set<User> actual_attendees = event.getAttendees();
+        Set<User> expected_attendees = ATTENDEES;
```

Comment:
I think you can just directly put the constants in your assertions, and that wouldn't affect readability much

Reply:
yes that looks much better thank you